src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleLines = sourceString.split(';');
+  const styleObject = {};
+
+  styleLines.forEach((line) => {
+    const cleanedLine = line.trim();
+
+    if (cleanedLine) {
+      const [property, value] = cleanedLine.split(':');
+
+      if (property && value) {
+        styleObject[property.trim()] = value.trim();
+      }
+    }
+  });
```

Comment:
[CODE PRACTICE] - Instead of using a `forEach` loop to iterate over the `styleLines`, consider using `reduce` which is more suited for transforming an array into an object. This aligns with the principle of using iteration methods for such transformations.